documentation/book/assembly-kafka-mirror-maker-numstreams.adoc

```diff
@@ -0,0 +1,17 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-numstreams-{context}']
+
+= Number of consumer streams
+
+In order to increase the throughput in mirroring topics, it is possible to increase the number of consumer threads.
+More consumer threads will belong to the same configured "consumer group".
+The topic partitions will be assigned across these consumer threads which will consume messages in parallel.
```

Comment:
The consumer is assigned a subset of the partitions by virtue of being in a consumer group. Are you saying that those assigned partitions are further subdivided and assigned to threads? Or are you saying that each thread functions as a consumer within the consumer group partitioning. Imho it's not clear from this sentence which applies.